Library/Homebrew/cmd/doctor.rb

```diff
@@ -435,6 +443,7 @@ def check_access_usr_local
 
     You should probably change the ownership and permissions of /usr/local
     back to your user account.
+      sudo chown $(whoami):admin /usr/local
```

Comment:
I'm not sure we should do this. `/usr/local` is not exclusive for homebrew. We should respect other softwares living there who choose to use root as default permission.